src/Pages/components/Searchbar.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react'
+import { v4 as uuid } from 'uuid'
+import styled from 'styled-components/macro'
+import useEvents from '../EventMain/useEvents'
+import Input from './common/Input'
+
+export default function Searchbar({
+  handleEventSuggestion,
+  handleEventSearch,
+}) {
+  const { eventArray } = useEvents()
+  const eventSuggestions = eventArray.map((event) => event.name)
+  const [filteredSuggestions, setFilteredSuggestions] = useState([])
+  const [eventNameInput, setEventNameInput] = useState('')
+
+  return (
+    <>
+      <EventSearchbar
+        placeholder="search..."
+        value={eventNameInput}
+        onKeyUp={handleInputKeyUp}
+        onChange={(e) => setEventNameInput(e.target.value)}
```

Comment:
In some functions you use "event" and in others the short "e". I would recommended to always use the same name. "event" may be a little bit easier to understand.